projects/vdk-plugins/vdk-notebook/src/vdk/plugin/notebook/notebook.py

```diff
@@ -87,14 +88,12 @@ def register_notebook_steps(file_path: Path, context: JobContext):
             context.step_builder._StepBuilder__steps.sort(key=lambda step: step.name)
             log.debug(f"{len(notebook_steps)} " f"cells with vdk tag were detected!")
         except json.JSONDecodeError as e:
-            errors.log_and_rethrow(
-                to_be_fixed_by=errors.ResolvableBy.USER_ERROR,
-                log=log,
-                what_happened=f"Failed to read the {file_path.name} file.",
-                why_it_happened=f"The provided {file_path.name} cannot be loaded into json format and "
-                f"cannot be read as a Jupyter notebook",
-                consequences=errors.MSG_CONSEQUENCE_TERMINATING_APP,
-                countermeasures=f"Check the {file_path.name} format again",
-                exception=e,
-                wrap_in_vdk_error=True,
+            errors.report_and_throw(
```

Comment:
@DeltaMichael I will leave it like that since, it is hard to navigate through the steps in the notebook and not everything is being executed. So I think adding additional context is good here

Reply:
i will be resolving those, but we can check those again after for the beta version of the integration, if we can introduce a better way of navigation through the notebook code